qiita_db/meta_util.py

```diff
@@ -37,11 +37,18 @@
 from re import sub
 from json import loads, dump, dumps
 
-from qiita_db.util import create_nested_path
+from qiita_db.util import create_nested_path, _retrieve_resource_data
```

Comment:
If `_retrieve_resource_data` is going to be use outside of util, then it should not start with _, could you change?